lib/std/target.zig

```diff
@@ -277,13 +277,13 @@ pub const Target = struct {
                         .aarch64 => VersionRange{
                             .semver = .{
                                 .min = .{ .major = 11, .minor = 6, .patch = 6 },
-                                .max = .{ .major = 12, .minor = 4 },
+                                .max = .{ .major = 13, .minor = 0 },
                             },
                         },
                         .x86_64 => VersionRange{
                             .semver = .{
                                 .min = .{ .major = 10, .minor = 15, .patch = 7 },
```

Comment:
Apple only supports the last 3 versions. This would mean this should be updated to:
```suggestion
                                .min = .{ .major = 11, .minor = 7, .patch = 1 },
```

Reply:
Andrew and I were discussing this and we were not sure if we want to remove Catalina support or not. It is true that Apple officially supports the last 3 versions of their OS but leaving Catalina support costs us only +3MB net size, so perhaps we could leave it in? I mean our linker will always support it now, and any new tweaks that works on newer macOS versions will be target-gated anyhow (like fixup chains, etc.). Dunno, food for thought, I'm happy either way.

Reply:
I will let @andrewrk have the final say on this and will wait on his decision before merging this :-)

Reply:
Sounds like the cost of supporting it is negligible then. I'd vote for keeping it as it is then and support Catalina.

Reply:
Hmmmm. Interesting choice.

Status quo policy is that we keep exactly 3 versions - dropping the oldest when we add the newest. My gut instinct is to stick with this, to keep our installation size stable, but even more importantly, to reduce scope of the project so that we can focus on more important things than looking after Apple's abandoned junk.

Here are two factors to consider:

1. When Zig gets closer to 1.0, and we have other big projects finished, we can consider increasing the scope of version ranges for Windows and macOS, because we will have more effort available to spend on it.
2. The universal-headers project is seeing some activity right now. If it is successful it may offer a way for us to retain a much higher number of versions of headers at very little cost in installation size.

So here's my conclusion. I'd like to drop Catalina. However, I'm giving override power to @Luukdegram - so if you're feeling strongly about it, you can make the call to keep Catalina and I will respect that.

Reply:
I am more than happy with that!

Reply:
Considering my opinion to support it was swayed so easily, I don't think it's strong enough to override your conclusion. My arguments to support it were solely based on minimal effort and it being nice to have ™. In that perspective, I believe your arguments against supporting it are much stronger so dropping the support is probably the sensible thing to do.

Reply:
Removing Catalina support then. Thank you both!